### PR TITLE
Fix: Fixed network locations not appearing on the sidebar

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...

Since becoming a view model, UpdateDrivesAsync() of NetworkDrivesViewModel was not being called anymore.

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Connect a network resource (e.g ftp server)
   2. Verify that it appears in the sidebar

**Screenshots (optional)**
Add screenshots here.
image

### CODE REVIEW

File: src/Files.App/Helpers/FtpHelpers.cs [hunk 1]
```diff
@@ -67,13 +67,9 @@ public static ushort GetFtpPort(string path)
...
 		{
 			path = path.Replace("\\", "/", StringComparison.Ordinal);
-			var schemaIndex = path.IndexOf("://", StringComparison.Ordinal) + 3;
-			var hostIndex = path.IndexOf("/", schemaIndex, StringComparison.Ordinal);
-
-			if (hostIndex == -1)
-				hostIndex = path.Length;
-
-			return path.Substring(schemaIndex, hostIndex - schemaIndex);
+			if (Uri.TryCreate(path, UriKind.Absolute, out var uri))
```
Comment (gave92) [inline src/Files.App/Helpers/FtpHelpers.cs:70]: Ftp path can contain username and pw: ftp://user@192.168.1.15:21/. Use Uri class to parse the Host/Authority as it supports user info unlike previous code.

**Comment by ferrariofilippo** [comment]
Shouldn't `Network` folder be the first entry?

**Comment by gave92** [comment]
It should but since #10258 drives get re-ordered in SidebarViewModel. Don't think it was intentional for network drives, I'll add a fix here.

**Review by ferrariofilippo — APPROVED** [review]
LGTM and works for me!

**Pull request merged**